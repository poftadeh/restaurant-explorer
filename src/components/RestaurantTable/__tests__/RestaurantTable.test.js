import React from 'react';
import { shallow } from 'enzyme';
import { RestaurantTable } from '../index';

it('renders <RestaurantTable /> properly', () => {
  const wrapper = shallow(<RestaurantTable />);
  expect(wrapper).toMatchSnapshot();
});
