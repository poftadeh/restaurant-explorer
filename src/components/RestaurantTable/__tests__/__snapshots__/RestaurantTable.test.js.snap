// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <RestaurantTable /> properly 1`] = `""`;
